client/src/components/InfoContainer.jsx

```diff
@@ -53,7 +53,7 @@ const InfoContainer = ({ season, path }) => {
             _hover={{
                 boxShadow: '0 5px 10px rgba(0,0,0,0.1), 0 3px 3px rgba(0,0,0,0.12);',
             }}
-            onClick={() => router.push(window.location.pathname + `/${path}/grades`)}
+            onClick={() => router.push(window.location.pathname + `/${path}`)}
```

Comment:
does this break any other pages? What happens when you click on a season now? Does it take you to /seasonId/grades or just /seasonId?

Reply:
This is actually a fix since /grades is already passed as a path to the InfoContainer as ``${season._id}/grades`` in the seasons page